### PR TITLE
Install empty OpenLDAP server

### PR DESCRIPTION
**Pull request by ReadyPlayerNaN**
Installs OpenLDAP server with empty database configured to be used with users and services.

### CODE REVIEW

**Comment by spetex** [comment]
@just-paja Could you please add a tiny bit of information about the module to readme? I think just few sentences about setup and running, otherwise LGTM.

**Comment by just-paja** [comment]
@spetex, added usage and example description to README.md

File: roles/ldap/meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+dependencies:
+  - { role: common, use_ldap: true }
```
Comment (spetex) [inline roles/ldap/meta/main.yml]: This is great! I didnt knew you can do this.

**Review by spetex — APPROVED** [review]
GJ

**Pull request merged**